application.properties

```diff
@@ -0,0 +1,6 @@
+#Setting the count of user name and password.
+user_name=
```

Comment:
Write some sentence to remember that here we need put values E.g. "user_name=Put user_name"

Reply:
Please in the next review a correct that.

Reply:
Please dear rail this is a step that a considerate not relevant but in the next review a going implement that.